slam-ws/transform/launch/setup_tf.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+    <arg name="model" default="$(find caffeine_description)/urdf/caffeine.urdf.xacro"/>
```

Comment:
Neither the arg nor the param seem to be used anywhere

Reply:
I saw your comment on the wiki: `NOTE: we specify robot_description as a top level param such that is available to both joint_state_publisher and robot_state_publisher.`. Sounds good